src/Files.App/Views/MainPage.xaml

```diff
@@ -150,7 +156,6 @@
 								VerticalAlignment="Bottom"
 								AutomationProperties.Name="{x:Bind Commands.NewTab.AutomationName}"
 								Background="Transparent"
-								BorderBrush="Transparent"
```

Comment:
I'm pretty sure this needs to be defined, please revert for now.

Reply:
BorderThinkess is already set 0 and so there's no effect. In spite of that, should?

Reply:
Is there a VisualState? I remember it was here for a reason but it's possible that it no longer applies.

Reply:
There isn't. Totally validated.

Reply:
Alright then, thanks for checking!